static/js/answers-experience.js

```diff
@@ -0,0 +1,51 @@
+import RuntimeConfig from './runtime-config';
+
+/**
+ * Check if Answers is initialized in window in a timed loop
+ * 
+ * @returns {Promise} When resolved, Answers exists in window
+ */
+function waitForAnswersInit() {
+  return new Promise((resolve, reject) => {
+    let timeElapsed = 0;
+    const duration = 5000;
+    const intervalLength = 10;
+
+    const interval = setInterval(() => {
+      timeElapsed += intervalLength;
+      if (window.hasOwnProperty('ANSWERS')) {
+        clearInterval(interval);
+        resolve();
+      }
+      if (timeElapsed > duration) {
+        clearInterval(interval);
+        reject('Timed out waiting on Answers initialization. Unable to update Answer\'s configuration(s).');
+      }
+    }, intervalLength);
+  });
+}
+
+/**
+ * Update Answer's runtime configurations
+ * 
+ * @param {Object} updatedConfig An object representation of the runtime config
+ */
+function updateAnswersConfig(updatedConfig) {
```

Comment:
It's a little bit strange to call this "updateAnswersConfig" when this doesn't actually update the internal state of the RuntimeConfig. In my head what this is really doing is listening for updates to a specific key, 'analyticsEventsEnabled', and adding a listener to update the SDK when such a change occurs. Would it be overkill to add the concept of listeners to RuntimeConfig?

Reply:
Ah it looks like there's already an _onUpdate() method, so maybe it's not so crazy to let you listen to a specific key?

Reply:
hmm i was thinking the naming implies updating configurations of answers object, not that it changes the runtimeconfig object of the site, if I'm understanding your comment right? But yeah we could change runtimeconfig to have individual listeners for keys so it doesn't need to run through all options on a single key/value change.

Reply:
updated to individual key listeners instead of all!